buildSrc/src/main/groovy/org/robolectric/gradle/GradleManagedDevicePlugin.groovy

```diff
@@ -0,0 +1,29 @@
+package org.robolectric.gradle
+
+import com.android.build.api.dsl.ManagedVirtualDevice
+import org.gradle.api.Plugin
+import org.gradle.api.Project
+
+class GradleManagedDevicePlugin implements Plugin<Project> {
+    @Override
+    void apply(Project project) {
+        project.android.testOptions {
+            devices {
```

Comment:
FYI: This has been moved to `project.android.testOptions.managedDevices.devices`.

Reply:
Hi @yuuki3655 , thanks for reviewing and suggestions. Are those changes are included from AGP 7.3? I have sent a draft PR to bump AGP to 7.3, and migrate some GMD properties to new spec: https://github.com/robolectric/robolectric/pull/7384. Could you help to review that PR? Thanks.